README.md

```diff
@@ -2,7 +2,7 @@
 
 | **Package Evaluator**                                           | **Build Status**                                                                                |
 |:---------------------------------------------------------------:|:-----------------------------------------------------------------------------------------------:|
-| [![0.4 package tests](http://pkg.julialang.org/badges/Geodesy_0.4.svg)](http://pkg.julialang.org/?pkg=Geodesy) [![0.5 package tests](http://pkg.julialang.org/badges/Geodesy_0.5.svg)](http://pkg.julialang.org/?pkg=Geodesy) | [![master build](https://travis-ci.org/JuliaGeo/Geodesy.jl.svg?branch=master)](https://travis-ci.org/JuliaGeo/Geodesy.jl) [![master coverage](http://img.shields.io/coveralls/JuliaGeo/Geodesy.jl.svg)](https://coveralls.io/r/JuliaGeo/Geodesy.jl) |
+| [![0.5 package tests](http://pkg.julialang.org/badges/Geodesy_0.5.svg)](http://pkg.julialang.org/?pkg=Geodesy) [![0.6 package tests](http://pkg.julialang.org/badges/Geodesy_0.6.svg)](http://pkg.julialang.org/?pkg=Geodesy) | [![Linux/macOS](https://travis-ci.org/JuliaGeo/Geodesy.jl.svg?branch=master)](https://travis-ci.org/JuliaGeo/Geodesy.jl) [![Windows](https://ci.appveyor.com/api/projects/status/github/NgYeeSian/Geodesy.jl?svg=true)](https://ci.appveyor.com/project/NgYeeSian/geodesy-jl) [![Coverage](http://img.shields.io/coveralls/JuliaGeo/Geodesy.jl.svg)](https://coveralls.io/r/JuliaGeo/Geodesy.jl) |
```

Comment:
It's hard to see but I removed the 0.4 package status badge. Should I keep this badge?